exporters/writers/hs_reduce_writer.py

```diff
@@ -48,14 +48,20 @@ def __init__(self, *args, **kwargs):
         self.logger.info('Will write accumulator to: {}#/details/{}'
                          .format(collection_url, self.element_key))
 
-    def write_batch(self, batch):
-        super(HubstorageReduceWriter, self).write_batch(batch)
-
+    def get_result(self, **extra):
         result = dict(self.reduced_result
                       if isinstance(self.reduced_result, MutableMapping)
                       else dict(value=self.reduced_result))
         result['_key'] = self.element_key
-        self.collection.set(result)
+        result.update(extra)
+        return result
+
+    def write_batch(self, batch):
+        super(HubstorageReduceWriter, self).write_batch(batch)
+        self.collection.set(self.get_result(finished=False))
```

Comment:
overall looks good to me, only thing is that it would be better to use `_finished` for the key name, to avoid conflicting with potential field names coming from `reduce_result`.